scene/gui/rich_text_label.cpp

```diff
@@ -5994,7 +6000,10 @@ void RichTextLabel::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("get_text"), &RichTextLabel::get_text);
 
+#ifndef DISABLE_DEPRECATED
 	ClassDB::bind_method(D_METHOD("is_ready"), &RichTextLabel::is_ready);
```

Comment:
I was thinking the same thing, actually. I just was not sure how things like GDExtension would've handled this

Reply:
`GDExtension` use the same method bind as GDScript and do not care about C++ code.

Reply:
It turns out that because internally `Node.is_ready()` is named this way internally, it somewhat conflicted with RichTextLabel's method too. The IDE is doing its best to understand it, so hopefully I have understood it right, too.